### DISCUSSION TITLE
Using Proposed API (textSearchProvider)

### DISCUSSION THREAD

**Discussion by ritikramuka** [microsoft/vscode-discussions#964] [Extension Development QnA] [answered]
I was trying to implement textSearchProvider API, in our extension.
but when I was doing ```npx vscode-dts main``` and overwriting the existing @types/vscode file, the vscode is unable to read any existing vscode exports and also any exports from textSearchProvider.d.ts

**Comment by isidorn** [comment, marked as answer]
These docs should help https://code.visualstudio.com/api/advanced-topics/using-proposed-api

Let us know if you are still hitting issues.
